app/services/load_db.py

```diff
@@ -0,0 +1,113 @@
+from pymodm import connect
+from app.db.mongo_model_definitions import *
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+
+class LoadDb:
+    def __init__(self):
+        pass
+
+    def __call__(self, data, db_name=os.getenv("MONGO_DB_ENDPOINT")):
+        connect(db_name)
+
+        services = [
+            Service(
+                id=s["id"],
+                name=s["name"],
+                description=s["description"],
+                tagline=s["tagline"],
+                countries=s["countries"],
+                related_services=s["related_service_ids"],
+                required_services=s["required_service_ids"],
```

Comment:
I'm not 100% sure (i don't lnow the specific format of `data`
But all above lines could be rewritten as:

```
Service(**s)
```

I don't know about nested objects, but there should be a better way to do this.

But if there isn't it can stay as is